web/src/beta/lib/core/StoryPanel/hooks.ts

```diff
@@ -61,6 +63,19 @@ export default (
     [selectedPageId, isEditable],
   );
 
+  const handlePageTime = useCallback(
+    (page: StoryPage) => {
+      const timePointField = page.property?.timePoint;
+      const currentTime = timePointField?.timePoint?.value;
+      if (!currentTime) onTimeChange?.(new Date());
+      else {
+        const getNewDate = new Date(currentTime.substring(0, 19)).getTime();
+        return onTimeChange?.(new Date(getNewDate));
```

Comment:
Is this correct? Why do you have to pass `getNewDate` in to another `new Date()` when you created getNewDate with a `new Date()`? If you don't do `getTime()` won't you have what you want?